src/supplemental/quic/quic_api.c

```diff
@@ -1455,6 +1457,7 @@ quic_pipe_open(void *qsock, void **qpipe, void *mqtt_pipe)
 int
 quic_pipe_close(void *qpipe, uint8_t *code)
 {
+	(void) code;
```

Comment:
Maybe u need this
`NNI_ARG_UNUSED(code)`

Reply:
Yes. u are right